akka-http/src/main/scala/akka/http/scaladsl/unmarshalling/sse/LineParser.scala

```diff
@@ -34,6 +34,7 @@ private final class LineParser(maxLineSize: Int) extends GraphStage[FlowShape[By
       import shape._
 
       private var buffer = ByteString.empty
+      private var slicedCr = false
```

Comment:
Maybe we should call it `lastCharWasCr` everywhere then it's pretty self-explanatory.

Reply:
Agree, this sounds much better. I'll change it.

Reply:
Done!